src/sgame/sg_main.cpp

```diff
@@ -2741,12 +2741,6 @@ void G_RunFrame( int levelTime )
 				G_FreeEntity( ent );
 				continue;
 			}
-			else if ( ent->unlinkAfterEvent )
-			{
-				// items that will respawn will hide themselves after their pickup event
```

Comment:
I would keep this as this can prove useful for mods. In fact, I was looking for a way to do that for the zombie mod and the juggernaut mod (where bases would be replaced by loots here and there)

Otherwise, looks good to me

Reply:
This is a bit of dead code left over from Quake. You can hardly expect that it is going to work or do what you want.